scipy/stats/_qmc.py

```diff
@@ -1894,3 +1874,35 @@ def _validate_workers(workers: IntNumber = 1) -> IntNumber:
                          "or > 0")
 
     return workers
+
+
+def _validate_bounds(
```

Comment:
I checked that nothing important changed since my last review:
<img width="940" alt="image" src="https://user-images.githubusercontent.com/6570539/160227977-580cef26-b34a-4058-80fe-602aee3d3426.png">